### PR TITLE
VZ-9739 Update the MySQL chart version and images

### PR DESCRIPTION
**Pull request by davisgomes**
**Changes**
- updated the MySQL and MySQL operator charts to the latest version 2.0.9
- updated the MySQL images to the latest version 8.0.33
- made updates to include previous version helm chart changes into the MySQL chart

**Testing**
- Installed locally on a single cluster to ensure that MySQL and keycloak get installed correctly

### CODE REVIEW

File: platform-operator/thirdparty/charts/mysql-operator/charts/.gitignore
Comment (fredtibbitts) [inline platform-operator/thirdparty/charts/mysql-operator/charts/.gitignore:1]: There are a few of these `.gitignore` files that are empty in the charts. Did these come from the upstream like this or did we add them? If we added them we should probably delete them (the empty ones).

File: platform-operator/thirdparty/charts/mysql/templates/deployment_cluster.yaml [hunk 1]
```diff
@@ -321,7 +321,7 @@ spec:
     {{- fail "Dump and Clone are mutually exclusive" }}
   {{- end }}
 
-  {{- if and .Values.initDB .Values.initDB.clone }}
+  {{- if (((.Values).initDB).clone) }}
```
Comment (jmaron99) [inline platform-operator/thirdparty/charts/mysql/templates/deployment_cluster.yaml:324]: Is this format preferred to the previous format?

**Comment by jmaron99** [comment]
did you test some upgrade scenarios?  1.5 to 1.6 or the upgrade paths pipeline?

**Review by fredtibbitts — APPROVED** [review]

File: platform-operator/thirdparty/charts/mysql/templates/deployment_cluster.yaml (hunk 1 above)
Comment (davisgomes) [reply platform-operator/thirdparty/charts/mysql/templates/deployment_cluster.yaml:324]: It is part of the chart update, so I'm assuming so

File: platform-operator/thirdparty/charts/mysql-operator/charts/.gitignore
Comment (davisgomes) [reply platform-operator/thirdparty/charts/mysql-operator/charts/.gitignore:1]: Those came from upstream. I's a bit weird, but I did not want to deviate much

**Review by mgianatagh — APPROVED** [review]

**Review by jmaron99 — APPROVED** [review]
LGTM but please don't merge till some of the periodic and push triggered scenarios validate functionality.

**Pull request merged**